templates/jenkins/files/README.md

```diff
@@ -1,19 +1,24 @@
-# Jenkins template for Platform.sh
+# Jenkins for Platform.sh
 
 This project provides a starter kit for Jenkins projects hosted on [Platform.sh](http://platform.sh).
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Jenkins is an open source automation server written in Java. Jenkins helps to automate the non-human part of the software development process, with continuous integration and facilitating technical aspects of continuous delivery.
 
-## Platform.sh configuration files
+## Customizations
+
+The following files and additions make the framework work.  If using this project as a reference for your own existing project, replicate the changes below to your project.
```

Comment:
"make Jenkins work".  It's not really a Framework the way Spring, Rails, or Symfony are.